flink-runtime-web/src/main/java/org/apache/flink/runtime/webmonitor/metrics/MetricStore.java

```diff
@@ -24,8 +24,8 @@
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
-import java.util.HashMap;
-import java.util.HashSet;
+import java.util.concurrent.ConcurrentHashMap;
+import java.util.concurrent.ConcurrentSkipListSet;
```

Comment:
Hi, @nicochen . Thank you for the `PR`. There is a import order problem, you should change the order of those import as the following code. Otherwise it will not pass the `checkstyle` system.
```java
import java.util.Map;
import java.util.Set;
import java.util.concurrent.ConcurrentHashMap;
import java.util.concurrent.ConcurrentSkipListSet;
```